src/ServiceControl.Audit/Infrastructure/CreateEventSource.cs

```diff
@@ -1,12 +1,11 @@
-﻿namespace ServiceControl.Audit.Infrastructure.Installers
+﻿namespace ServiceControl.Audit.Infrastructure
 {
     using System.Diagnostics;
     using System.Threading.Tasks;
-    using NServiceBus.Installation;
 
-    public class CreateEventSource : INeedToInstallSomething
+    class EventSource
     {
-        public Task Install(string identity)
+        public Task Create()
```

Comment:
any reason to not make this static (and `void`)?

Reply:
No special reason. I had it static at first but with the task. Then I thought about asking if we have any preference for the static part. For the void part, it makes sense to make it so.